src/components/dropdowns/BaseDropdownSelectQuickSwapTokeList.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+    <b-dropdown variant="link" class="dropdown-select">
+        <template #button-content>
+            <div v-if="erc20Token && erc20Token.chainId === chainId">
+                <div class="d-flex align-items-center">
+                    <img
+                        v-if="!erc20Token._id"
+                        :src="erc20Token.logoURI"
+                        class="mr-3"
+                        width="20"
+                        :alt="erc20Token.name"
+                    />
+                    <base-identicon v-else class="mr-3" :size="20" variant="darker" :uri="erc20Token.logoURI" />
+                    <strong class="mr-1">{{ erc20Token.symbol }}</strong> {{ erc20Token.name }}
+                </div>
+            </div>
+            <div v-else>Select an ERC20 token</div>
+        </template>
+        <b-dropdown-item v-if="!hasERC20s"> No tokens available. </b-dropdown-item>
+
+        <b-dropdown-item-button :key="erc20.address" v-for="erc20 of tokenList" @click="onTokenListItemClick(erc20)">
+            <img :src="erc20.logoURI" width="20" class="mr-3" :alt="erc20.name" />
+            <strong>{{ erc20.symbol }}</strong> {{ erc20.name }}
+        </b-dropdown-item-button>
+    </b-dropdown>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+import axios from 'axios';
+import { TERC20 } from '@/types/erc20';
+import BaseIdenticon from '../BaseIdenticon.vue';
+import { ChainId } from '@/types/enums/ChainId';
+
+const QUICKSWAP_TOKEN_LIST =
+    'https://unpkg.com/quickswap-default-token-list@1.2.40/build/quickswap-default.tokenlist.json';
+
+@Component({
+    components: {
+        BaseIdenticon,
+    },
+})
+export default class DropDownSelectQuickSwapTokeList extends Vue {
```

Comment:
@valeriagrazzini  Let's rename this to BaseDropDownSelectERC20, then it will be a bit more future proof. We might deviate from using that quickswap list if we find a better source for ERC20 info.

Reply:
@peterpolman It's ok but that name is already taken by the main list which contains the quickswap list and the tokens created by the user

Reply:
Let's make it `BaseDropDownSelectPolygonERC20` then! Since it only displays Polygon ERC20s anyway.